modules/margin/src/main/java/com/opengamma/sdk/margin/v3/MultiCcpMarginCalcResult.java

```diff
@@ -0,0 +1,543 @@
+/*
+ * Copyright (C) 2017 - present by OpenGamma Inc. and the OpenGamma group of companies
+ *
+ * Please see distribution for license.
+ */
+package com.opengamma.sdk.margin.v3;
+
+import java.time.LocalDate;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashMap;
+import java.util.List;
+import java.util.Map;
+import java.util.NoSuchElementException;
+import java.util.Optional;
+
+import org.joda.beans.ImmutableBean;
+import org.joda.beans.JodaBeanUtils;
+import org.joda.beans.MetaBean;
+import org.joda.beans.MetaProperty;
+import org.joda.beans.TypedMetaBean;
+import org.joda.beans.gen.BeanDefinition;
+import org.joda.beans.gen.PropertyDefinition;
+import org.joda.beans.impl.direct.DirectFieldsBeanBuilder;
+import org.joda.beans.impl.direct.MinimalMetaBean;
+
+@BeanDefinition(constructorScope = "private", metaScope = "private", factoryName = "of", style = "minimal")
+public final class MultiCcpMarginCalcResult implements ImmutableBean {
+  /**
+   * The status of the calculation - PENDING or COMPLETED.
+   */
+  @PropertyDefinition(validate = "notNull")
+  private final MarginCalcResultStatus status;
+  /**
+   * The type of calculation.
+   */
+  @PropertyDefinition(validate = "notNull")
+  private final MarginCalcRequestType type;
+  /**
+   * The valuation date for which the portfolio will be processed.
+   */
+  @PropertyDefinition(validate = "notNull")
+  private final LocalDate valuationDate;
+  /**
+   * The reporting currency, as an ISO 4217 three letter currency code.
+   */
+  @PropertyDefinition(validate = "notNull")
+  private final String reportingCurrency;
+
+  @PropertyDefinition(validate = "notNull")
+  private final boolean applyClientMultiplier;
+  /**
+   * The summary of the portfolio items, may be empty.
+   */
+  @PropertyDefinition(validate = "notNull")
+  private final List<PortfolioItemSummary> portfolioItems;
+
+  /**
```

Comment:
Try to be consistent - either have a blank line between the fields, or not. I prefer not :-)